projects/demo/src/app/pages/font/examples/1/index.less

```diff
@@ -1,38 +1,40 @@
-.text-h1,
-.text-h2,
-.text-h3,
-.text-h4,
-.text-h5,
-.text-h6 {
-    margin: 0.3125rem 0;
-}
+::ng-deep {
```

Comment:
:host ::ng-deep? Otherwise once you open this example these fonts will become available and applied all across the app

Reply:
fixed